PSReadLine/ReadLine.cs

```diff
@@ -46,16 +46,16 @@ public partial class PSConsoleReadLine : IPSConsoleReadLineMockableMethods
 
         private bool _delayedOneTimeInitCompleted;
 
-        private IPSConsoleReadLineMockableMethods _mockableMethods;
-        private IConsole _console;
+        private readonly IPSConsoleReadLineMockableMethods _mockableMethods;
+        private readonly IConsole _console;
         private ICharMap _charMap;
         private Encoding _initialOutputEncoding;
         private bool _skipOutputEncodingChange;
         private EngineIntrinsics _engineIntrinsics;
         private Thread _readKeyThread;
         private AutoResetEvent _readKeyWaitHandle;
         private AutoResetEvent _keyReadWaitHandle;
-        private AutoResetEvent _forceEventWaitHandle;
+        private readonly AutoResetEvent _forceEventWaitHandle;
```

Comment:
These 3 should be reverted. PSReadLine xUnit test needs to set `_mockableMethods` and `_console` via reflection, and PSES needs to set `_forceEventWaitHandle` via reflection.